src/warden/src/bin/reftest.rs

```diff
@@ -239,7 +239,7 @@ fn main() {
         println!("Warding GL headless:");
         let events_loop = glutin::EventsLoop::new();
         let context =
-            glutin::Context::new(&events_loop, glutin::ContextBuilder::new(), false).unwrap();
+            glutin::Context::new_headless(&events_loop, glutin::ContextBuilder::new(), glutin::dpi::PhysicalSize::new(0.0, 0.0)).unwrap();
```

Comment:
@kvark Not sure if this zero-sized context is appropriate

Reply:
did you run reftests with it?

Reply:
Yes, but see screenshot. I am not sure what to make of it.

Reply:
is this matching what you get without this PR?

Reply:
yes